models/classes/TestModelService.php

```diff
@@ -53,7 +55,8 @@ public function prepareContent( \core_kernel_classes_Resource $test, $items = ar
      * {@inheritDoc}
      * @see \taoTests_models_classes_TestModel::deleteContent()
      */
-    public function deleteContent( \core_kernel_classes_Resource $test) {
+    public function deleteContent(\core_kernel_classes_Resource $test)
```

Comment:
Add to import section

```suggestion
    public function deleteContent(core_kernel_classes_Resource $test)
```